src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -490,6 +493,85 @@ struct uct_ib_mlx5_query_hca_cap_in_bits {
     uint8_t    reserved_at_40[0x40];
 };
 
+struct uct_ib_mlx5_hca_vport_context_bits {
+    uint8_t    field_select[0x20];
+
+    uint8_t    reserved_at_20[0xe0];
+
+    uint8_t    sm_virt_aware[0x1];
+    uint8_t    has_smi[0x1];
+    uint8_t    has_raw[0x1];
+    uint8_t    grh_required[0x1];
+    uint8_t    reserved_at_104[0xc];
+    uint8_t    port_physical_state[0x4];
+    uint8_t    vport_state_policy[0x4];
+    uint8_t    port_state[0x4];
+    uint8_t    vport_state[0x4];
+
+    uint8_t    reserved_at_120[0x20];
+
```

Comment:
what is the logic to add extra line or not?

Reply:
it is copied from mlx5, i've preserved the same alignment